actionview/test/template/form_helper_test.rb

```diff
@@ -3256,6 +3290,18 @@ def test_fields_for_with_index
     assert_dom_equal expected, output_buffer
   end
 
+  def test_fields_for_with_index_field_id
+    fields_for(:post, @post) do |fields|
+      assert_equal "post_title", fields.field_id(:title)
```

Comment:
This test is currently identical to `test_fields_for_field_id`. I guess it was meant to be something like this:

```suggestion
    fields_for(:post, @post, index: 1) do |fields|
      assert_equal "post_1_title", fields.field_id(:title)
```